libc-bottom-half/headers/public/wasi/api.h

```diff
@@ -0,0 +1,2569 @@
+/**
+ * THIS FILE IS AUTO-GENERATED!
```

Comment:
By whom?  Where I go if I want to modify it?

Reply:
I've now added a patch to add more information here.